eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdin$EOnext_line.java

```diff
@@ -59,7 +61,17 @@ public class EOstdin$EOnext_line extends PhDefault {
                                 "There is no line in the standard input stream to consume"
                             );
                         }
-                        return new Data.ToPhi(sc.nextLine());
+                        final String line = sc.nextLine();
```

Comment:
@includealex Maybe it would be better to use buffered `string` from `EOstdin$EOφ`? Like this:

```
final Phi stdin = rho.attr("ρ").get();
final String input = new Param(stdin, "Δ").strong(String.class);
final String[] lines = input.split(System.lineSeparator());
```

Reply:
@Graur thanks, good idea. But this line
```java
final String input = new Param(stdin, "Δ").strong(String.class);
```
calls stdin, that will call System.in every time when we use it. And we are facing the same problem we are trying to fix. That is why  file `EOstdin$EOφ` should be changed too.